### DISCUSSION TITLE
Where to store APPLE_API_KEY & APPLE_API_ISSUER & APPLE_API_KEY_PATH

### DISCUSSION THREAD

**Discussion by Sembauke** [tauri-apps/tauri#11484] [Q&A] [answered]
Currently I am storing my `APPLE_API_KEY & APPLE_API_ISSUER & APPLE_API_KEY_PATH` in my .env file. But it seems not to be the right place to store them.

Where should I store these values?

**Comment by FabianLars** [comment, marked as answer]
Tauri cannot read .env files so for the cli to pick those up for code signing you need to set actual env vars.

For the current terminal session you can do that by typing `export APPLE_API_KEY="abc"` into the terminal and to store that permanently (not reallyyy recommended because then all programs can always access those) you could add those same export statements into `~/.zshrc` or `~/.zprofile`.

If you don't want to write them into the zsh file you could store the vars in your password manager for example, or i guess you can also keep it in the .env file as long as you do not commit it into git and upload it.
